daemons/execd/pacemaker-execd.c

```diff
@@ -551,5 +557,11 @@ main(int argc, char **argv, char **envp)
 
     /* should never get here */
```

Comment:
Why not? Seems like the mainloop would exit during a normal shutdow

Reply:
We always go through lrmd_exit(), which calls crm_exit(). It might be better for lrmd_exit() to just drain and quit the mainloop, then handle all the cleanup here. But not necessary for this PR.

Reply:
Oh I mean the comment suggests that we should never get to the call of `lrmd_exit()`... maybe just out of place?

Reply:
No, the comment's correct. Any path out of the mainloop already calls lrmd_exit() which calls crm_exit(), so we'll never quit the mainloop or reach the lrmd_exit() call here.

Reply:
Oh, okay